packages/now-node/src/dev-server.ts

```diff
@@ -0,0 +1,54 @@
+import http from 'http';
+import path from 'path';
+import { createServerWithHelpers } from './helpers';
+
+function listen(
+  server: http.Server,
+  port: number,
+  host: string
+): Promise<void> {
+  return new Promise(resolve => {
+    server.listen(port, host, () => {
+      resolve();
+    });
+  });
+}
+
+async function main() {
+  const entrypoint = process.env.NOW_DEV_ENTRYPOINT;
+  if (!entrypoint) {
+    throw new Error('`NOW_DEV_ENTRYPOINT` must be defined');
+  }
+
+  //const shouldAddHelpers = true;
+
+  const entrypointPath = path.join(process.cwd(), entrypoint);
+  const handler = await import(entrypointPath);
+
+  /*
+  const server = http.createServer((req, res) => {
+    Promise.resolve(true).then(() => handler.default(req, res)).catch(err => {
+      console.error('Caught error from HTTP handler:', err);
+      if (!res.headersSent) {
+        res.statusCode = 500;
+        res.end('Internal server error\n');
+      }
+    });
+  });
+  */
+  const server = createServerWithHelpers(handler.default);
```

Comment:
I think you need to make this conditional based on `shouldAddHelpers`.

Reply:
Also is `default` guaranteed to exist?

Reply:
> I think you need to make this conditional based on `shouldAddHelpers`.

See the TODO list.

> Also is `default` guaranteed to exist?

For JS files, yes. But for TS, I suppose there's a chance that the user doesn't `export default`, in which case an error message should be printed. Will update.

Reply:
I was thinking we need something like the following from `launcher.ts`:

```js
let listener = require(entrypointPath);
if (listener.default) listener = listener.default;
```

Reply:
I don't think so, because `await import()` adds some wrapper logic to `module.exports` be returned as `default` for JS modules.